### PR TITLE
Added missing wxTextCtrl accelerators.

### PR DESCRIPTION
**Pull request by owenwengerd-octave**
These were added to wxTextCtrl in our fork several years ago, before the handling was moved to wxTextEntry.

### CODE REVIEW

File: src/msw/textentry.cpp [hunk 1]
```diff
@@ -1123,13 +1123,24 @@ bool wxTextEntry::MSWShouldPreProcessMessage(WXMSG* msg) const
...
                                 return false;
                         }
                     }
                     else // Shift is pressed
                     {
-                        if ( vkey == VK_INSERT || vkey == VK_DELETE )
-                            return false;
+                        switch ( vkey )
+                        {
+                            case ' ':
```
Comment (vadz) [inline src/msw/textentry.cpp:1135]: Does `Shift-Space` have some special meaning for the text control? AFAICS it doesn't do anything special in it, so why should we have it here?

**Review by vadz — COMMENTED** [review]
Thanks, I agree that the arrows (and `Shift-{Home,End}`) should still work in the text control even if they're used as accelerators (although I wonder if we shouldn't handle them at wxWindow level by examining whether `WM_GETDLGCODE` returns `DLGC_WANTARROWS`), but I'm not sure about `Shift-Space`, could you please explain why is this needed? TIA!

**Comment by owenwengerd-hexagon** [comment]
I could not find in our records a reason for adding Shift-Space. I see there is mention at https://trac.wxwidgets.org/ticket/17870 about Space as an accelerator, but the ones we added appear more likely to come from either manual testing, or by scanning the wxTextCtrl source for keys that get special treatment.

File: src/msw/textentry.cpp (hunk 1 above)
Comment (owenwengerd-hexagon) [reply src/msw/textentry.cpp:1135]: I assume that we wanted Shift-Space to result in a normal space character even when a parent window interprets Shift-Space as an accelerator. Apparently Ctrl-Space is fine to be interpreted as an accelerator.

**Comment by vadz** [comment]
But I don't think space gets special treatment, or at least nothing more special than any other alphanumeric character. So I think it doesn't make sense to handle it specially, because if we do, we should also handle all the other printable keys as well.

Or is there some reason space is different that I'm still missing?

File: src/msw/textentry.cpp (hunk 1 above)
Comment (owenwengerd-hexagon) [reply src/msw/textentry.cpp]: I agree, special handling for `Shift-Space` was a mistake. I've removed it.

**Comment by vadz** [comment]
Thanks for the update! Will merge in a moment.

**Pull request closed without merging**